GithubBrowserSample/app/src/main/java/com/android/example/github/repository/RepoRepository.java

```diff
@@ -131,7 +138,10 @@ protected boolean shouldFetch(@Nullable Repo data) {
     public LiveData<Resource<List<Contributor>>> loadContributors(String owner, String name) {
         return new NetworkBoundResource<List<Contributor>, List<Contributor>>(appExecutors) {
             @Override
-            protected void saveCallResult(@NonNull List<Contributor> contributors) {
+            protected void saveCallResult(@Nullable List<Contributor> contributors) {
+                if (contributors == null) {
```

Comment:
there are no tests covering the null case for any method that was updated. 
Please add tests.